### ISSUE TITLE
failed to parse config:  * Invalid label: No argument or block type is named "logging".

### ISSUE THREAD

**Issue by jamjon3** [hashicorp/nomad-driver-podman#154]
The deployment fails when I add in 'logging' into the config under task. It works when I remove it. What am I doing wrong here???

failed to parse config:
Feb 08 22:44:49 somevmnmc01 nomad[67242]:   |         * Invalid label: No argument or block type is named "logging".

The task looks something like this:

```task "someapp-deploy" {
  driver = "podman"
  config {
        image = "someimage"
        force_pull = true
        auth {
          username = "someuser"
          password = "somepassword"
        }
        logging = {
          driver = "journald"
          options = [
            {
              "tag" = "someapp"
            }
          ]
        }
  }

**Comment by towe75** [comment]
Are you building the driver from latest main branch? Seems like v0.3.0 binary did not yet have this feature.

**Comment by kamaradclimber** [comment]
This syntax is described in current documentation of the podman driver but seems to require an unreleased version of the driver.
Could you release the driver (or correct the documentation) please?

**Comment by jamjon3** [comment]
@towe75 I'm building against the 0.3.0 (latest release). The last release was in 2021 and I'm seeing a lot of changes and I hope we are close to a new release and I agree with @kamaradclimber to just cut a new release

**Comment by lgfa29** [comment]
Hi all, 

I believe this is the same problem as #170 which should be fixed in [v0.4.0](https://releases.hashicorp.com/nomad-driver-podman/0.4.0/). Would you mind trying with the new version?

I will also close this issue in favor of #170, so please post any updates there 🙂 

Thank you.
